content/articles/code-style.md

```diff
@@ -0,0 +1,113 @@
+Title: Python Code Style at We Are Wizards
+Date: 2015-01-02
+Category: Code
+Authors: Gautier
+Short_summary: I write down some of the style habits I have when when writing Python code at We Are Wizards
+
+
+Here at We Are Wizards, we code review most of our commits for most of our
+projects, this means readable and consistent code style helps get code
+merged and ready for deployment.
```

Comment:
that sentence above doesn't make much sense IMO - are you trying to say that the code getting merged is more readable?